### ISSUE TITLE
gl history invalid start byte

### ISSUE THREAD

**Issue by Azd325** [sdg-mit/gitless#37]
I use python 3.4
```
↪ gl history
# Oops...something went wrong (recall that Gitless is in beta). If you want to help, report the bug at http://gitless.com/community.html and include the following information:

0.6.2

Traceback (most recent call last):
  File "/home/tim/.virtualenvs/tempenv-7cc217326033a/lib/python3.4/site-packages/gitless/cli/gl.py", line 73, in main
    return SUCCESS if args.func(args) else ERRORS_FOUND
  File "/home/tim/.virtualenvs/tempenv-7cc217326033a/lib/python3.4/site-packages/gitless/cli/gl_history.py", line 30, in main
    for ci in repo_lib.history(include_diffs=args.verbose):
  File "/home/tim/.virtualenvs/tempenv-7cc217326033a/lib/python3.4/site-packages/gitless/core/repo.py", line 53, in history
    return git_log.log(include_diffs=include_diffs)
  File "/home/tim/.virtualenvs/tempenv-7cc217326033a/lib/python3.4/site-packages/gitpylib/log.py", line 26, in log
    log_fmt, '-p' if include_diffs else ''))
  File "/home/tim/.virtualenvs/tempenv-7cc217326033a/lib/python3.4/site-packages/gitpylib/common.py", line 37, in safe_git_call
    ok, out, err = git_call(cmd)
  File "/home/tim/.virtualenvs/tempenv-7cc217326033a/lib/python3.4/site-packages/gitpylib/common.py", line 50, in git_call
    out = out.decode('utf-8')
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xf6 in position 969175: invalid start byte
```

**Comment by spderosso** [comment]
Thanks for reporting this

**Comment by spderosso** [comment]
57259b7eddc940a6ec1fb02b312cd14fffd6d0ff

**Comment by spderosso** [comment]
Fix will go out in next release (0.7)
